app/models/user.rb

```diff
@@ -15,10 +13,15 @@ class User < ActiveRecord::Base
   validates :username, presence: true
   validates :email, presence: true, uniqueness: true
   validates :gender, presence: true
+  validates_uniqueness_of :phone_number
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.